### DISCUSSION TITLE
str.replace

### DISCUSSION THREAD

**Discussion by snurulazizah** [teamalgoritma/community#155] [Q&A] [answered]
Cara assign dua str.replace kedalam satu variable gimana caranya ya?

**Comment by yosiaazarya** [comment, marked as answer]
Jika suatu kolom bernama `value` memiliki data seperti berikut: "IDR75,000"

Maka yang dapat kita lakukan adalah dengan cara ini: `df['value'].str.replace('IDR', '').str.replace(',' , '')`
